tests/h/conftest.py

```diff
@@ -195,6 +195,7 @@ def invalid_form(errors=None):
 
 @pytest.fixture
 def matchers():
+    # pylint: disable=redefined-outer-name
```

Comment:
So I guess the `matchers` fixture is still used, but now only for the h-specific matchers, which have been reimplemented using h-matchers but are still accessed via the `matchers` fixture

Reply:
Yep that's right. I didn't think these were general enough to share.